docs/_tutorials/create-jekyll-theme.md

```diff
@@ -147,7 +147,7 @@ Some page content here...
 Create another page for testing called `about.md` with similar front matter tags.
 
 {: .note .info}
-**Note:** If you don't specify a layout in your pages, Jekyll will automatically use the template labeled `default`. We specify it here only to make it explicit what's happening.
+If you don't specify a layout in your pages, Jekyll will simply render that page as an unstyled basic HTML page.
 
```

Comment:
Jekyll doesn't automatically assign a layout. GitHub Pages does it via a [plugin](https://github.com/benbalter/jekyll-default-layout).